client/src/app/core/core-services/active-meeting-id.service.ts

```diff
@@ -50,7 +50,26 @@ export class ActiveMeetingIdService {
         this.lifecycleService.openslidesShutdowned.subscribe(() => this.shutDown());
     }
 
-    public start(): void {
+    private start(): void {
```

Comment:
move start and shutdown directly to the constructor as lambdas, if they are not used somewhere else.

Reply:
Why? This makes the functions smaller than if everything is in the constructor...

Reply:
but they are only used there